internal/provider/data_source_gitlab_current_user_test.go

```diff
@@ -0,0 +1,38 @@
+//go:build acceptance
+// +build acceptance
+
+package provider
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/xanzy/go-gitlab"
+)
+
+func TestAccDataSourceGitlabCurrentUser_basic(t *testing.T) {
+	//The root user has no public email by default, set the public email so it shows up properly.
+	_, _, _ = testGitlabClient.Users.ModifyUser(1, &gitlab.ModifyUserOptions{
```

Comment:
I suggest that we add a `t.Cleanup` function to revert that back - especially for when we start to parallelise test execution.

Reply:
@timofurrer - Makes sense - I'll get that added. It's worth noting that "admin@example.com" is the _only_ value we could set here based on our current setup, so any other test would be setting it to exactly the same value